tools/mysql-replica-testing/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
       [
         "mysqld",
         "--datadir=/tmp/mysqldata-replica-main",
+        "--default-authentication-plugin=mysql_native_password",
```

Comment:
I was having troubles with the `replicator` user account created during the setup process without this.